gluten-core/src/main/scala/io/glutenproject/expression/ConverterUtils.scala

```diff
@@ -469,4 +476,17 @@ object ConverterUtils extends Logging {
   // Other
   final val ROW_CONSTRUCTOR = "row_constructor"
 
+  // The blacklist for Velox unsupported or mismatched expressions with specific input type,
```

Comment:
Can we move the configuration to Velox backends folder?

Reply:
Yes, I've moved the configuration to Velox backends.